app/store/configureStore.js

```diff
@@ -1,36 +0,0 @@
-import { createStore, applyMiddleware, compose } from 'redux';
-import { routerMiddleware } from 'react-router-redux';
-import thunk from 'redux-thunk';
-import rootReducer from 'reducers';
-import promiseMiddleware from 'middlewares/promiseMiddleware';
-import createLogger from 'redux-logger';
-
-/*
- * @param {Object} initial state to bootstrap our stores with for server-side rendering
- * @param {History Object} a history object. We use `createMemoryHistory` for server-side rendering,
```

Comment:
Hey I just had a look again. https://github.com/18601673727/react-webpack-node/tree/348472fb49c5fa402bb7764a6de45b3f443b508c/app

I think the `configureStore` was added in a separate commit.